etc/repo-config.yml

```diff
@@ -128,6 +128,14 @@ repos:
       - yum/redhat/8/mongodb-org
       - yum/redhat/8Server/mongodb-org
 
+  - name: rhel88
+    type: rpm
+    edition: org
+    bucket: repo.mongodb.org
+    repos:
+      - yum/redhat/8/mongodb-org
```

Comment:
This is all in the same repo for other minor versions- do they overwrite each other?

Reply:
I'm not sure, but the server repo does [the same thing](https://github.com/10gen/mongo/blob/master/etc/repo_config.yaml#L410-L416).